services/scabbard/src/cli/action/namespace.rs

```diff
@@ -0,0 +1,105 @@
+// Copyright 2020 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use clap::{App, Arg, ArgMatches, SubCommand};
+use sabre_sdk::protocol::payload::{
+    Action, CreateNamespaceRegistryActionBuilder, DeleteNamespaceRegistryActionBuilder,
+    SabrePayload, SabrePayloadBuilder, UpdateNamespaceRegistryOwnersActionBuilder,
+};
+
+use super::CliError;
+
+pub fn subcommand<'a, 'b>() -> App<'a, 'b> {
+    SubCommand::with_name("ns")
+        .about("create, update, or delete a Sabre namespace")
+        .subcommand(SubCommand::with_name("create").args(&generate_create_or_update_args()))
+        .subcommand(SubCommand::with_name("update").args(&generate_create_or_update_args()))
+        .subcommand(SubCommand::with_name("delete").arg(generate_namespace_arg()))
+}
+
+fn generate_create_or_update_args<'a, 'b>() -> Vec<Arg<'a, 'b>> {
+    vec![
+        generate_namespace_arg(),
+        Arg::with_name("owner")
+            .help("owner of this namespace")
```

Comment:
Help text should start with a capital letter to match --help/--version usage.

Reply:
Updated all `about`s and `help`s to start with capital letters